spec/enumerable_spec.rb

```diff
@@ -0,0 +1,108 @@
+require_relative '../lib/enumerable.rb'
+
+
+
+describe Enumerable do
+  let(:arr){[1,2,3,4,5]}
+
+  describe '#my_each' do
+    let(:arr){[1,2,3,4,5]}
```

Comment:
Having this line in your code makes it WET type of code, more details in my comment.
i comment out the line 9, and call arr on line 14, your code works fine and better with that, kindly apply to the rest .

Reply:
I left an additional note to tell the reviewer that the RSpec is not required for this project, so I do not understand why it's been reviewed. However I have removed it to avoid confusion.